utils/config.ts

```diff
@@ -15,14 +15,16 @@ import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
 
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
-// eslint-disable-next-line @typescript-eslint/ban-types
+
 export const IConfig = t.interface({
   AzureWebJobsStorage: NonEmptyString,
 
   COSMOSDB_KEY: NonEmptyString,
   COSMOSDB_NAME: NonEmptyString,
   COSMOSDB_URI: NonEmptyString,
 
+  IPA_OPENDATA_URL: NonEmptyString,
```

Comment:
@balanza my fault, it should be _IPA_OPEN_DATA_STORAGE_PATH_, because it is referred from [function.json](https://github.com/pagopa/io-selfcare-importadesioni/blob/IOPAE-130--sync-adesioni-e-contratti/OnContractChange/function.json#L18) (`"path": "%IPA_OPEN_DATA_STORAGE_PATH%",`) just to not harcoded the input path, but if you think it is unnecessary i can hardcode it

Reply:
Also `IPA_OPEN_DATA_STORAGE_PATH` isn't defined as environment variable in https://github.dev/pagopa/io-selfcare-importadesioni/blob/a388e47409b9e1e1c759d17d8f9066b1a89309b9/infra/resources/functions.tf#L65

What value do you expect it to have?

Reply:
fixed with 2e69ec71727818fe841b95980f5f4367ea58b9e2